src/mjolnir/ingest_transit.cc

```diff
@@ -523,6 +570,8 @@ bool write_stop_pair(Transit& tile,
       if (!feed.get_frequencies(currTrip->trip_id).empty()) {
         const auto& currFrequencies = feed.get_frequencies(currTrip->trip_id);
         if (currFrequencies.size() > 1) {
+          // TODO(nils): this should be properly handled as 1 trip id can have
+          // multiple frequencies, e.g. the example Google feed does
```

Comment:
yeah I confirmed there can be more than 1 frequency per trip id, see https://developers.google.com/static/transit/gtfs/examples/sample-feed.zip